chain-impl-mockchain/src/vote/manager.rs

```diff
@@ -117,6 +129,9 @@ pub enum VoteError {
 
     #[error("Error during private tallying {0}")]
     PrivateTallyError(String),
+
+    #[error("It's not possible to cast a vote if the tally is already decrypted")]
+    TallyAlreadyDecrypted,
```

Comment:
Is there a specific need for such error? Votes can be cast only during the vote period and can be decrypted during tally period.